about.html

```diff
@@ -12,52 +12,91 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <nav>
+    <img src="img/lambda-black.png" alt="lambda logo">
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
+    <a href="#products">Products</a>
+    <a href="#blog">Blog</a>
+    <a href="#contact">Contact</a>
+</nav>
 
+    <div class="container about-page">
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
+      <img src="img/jumbo-about.png" alt="skyline">
+     
+  <section class="top-about">
+    <div class="about-text">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+      <h2>About</h2>
 
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  
+  </div>
+  </section>
+    
+       
+<section class="middle-container">
+  <div class="strategy">
+    <img src="img/about-plan.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For accessibility purposes, it's recommended that the attribute reflect the sections more accurately.